server/src/main/java/org/opensearch/common/io/stream/StreamInput.java

```diff
@@ -643,6 +643,33 @@ public <K, V> Map<K, V> readMap(Writeable.Reader<K> keyReader, Writeable.Reader<
         return map;
     }
 
+    /**
```

Comment:
Not needed, right?

Reply:
Yep was just about to delete it, you beat me to it